evm-adapters/src/call_tracing.rs

```diff
@@ -575,3 +594,56 @@ fn diff_score(bytecode1: &[u8], bytecode2: &[u8]) -> f64 {
     // println!("diff_score {}", diff_chars as f64 / cutoff_len as f64);
     diff_chars as f64 / cutoff_len as f64
 }
+
+fn decode_solidity_reverts(data: &[u8]) -> Option<&str> {
+    if data.len() >= 4 {
+        match data[0..4] {
+            // keccak(Panic(uint256))
+            [78, 72, 123, 113] => {
+                // ref: https://soliditydeveloper.com/solidity-0.8
+                match data[data.len() - 1] {
+                    1 => {
+                        // assert
+                        Some("assert failure")
+                    }
+                    17 => {
+                        // safemath over/underflow
+                        Some("SafeMath over/underflow")
+                    }
+                    18 => {
+                        // divide by 0
+                        Some("Divide by 0")
```

Comment:
```suggestion
                        Some("Division by 0")
```

Reply:
Could also be when modulo by zero
```suggestion
                        Some("Division by 0 / modulo by zero")
```

Reply:
good point should def include that